src/common/c_jhash.h

```diff
@@ -24,7 +24,11 @@
 #ifndef _C_JHASH_H
 #define _C_JHASH_H
 
+#ifdef __cplusplus
 #include <cstdint>
+#else
+#include <stdint.h>
```

Comment:
I'd say just include stdint.h and add a // NOLINT comment so that it's not flagged by clang-tidy.

I did this at a couple other places, odd that this one didn't get caught by the CI... did we loose the building of the csync tests somehow?

Reply:
OK, I went with this suggestion.